diffblue.yml

```diff
@@ -3,3 +3,5 @@ testCmd: cd main-module; mvn test
 cbmcArguments:
   java-max-input-array-length: 10
   unwind: 10
+  refine-strings: true
```

Comment:
I suggest we don't add this here, but wait for platform to be updated with it instead